dependencies/vaticle/repositories.bzl

```diff
@@ -28,7 +28,7 @@ def vaticle_typedb_protocol():
     git_repository(
         name = "vaticle_typedb_protocol",
         remote = "https://github.com/typedb/typedb-protocol",
-        tag = "3.0.0-alpha-3",  # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_typedb_protocol
+        commit = "94b79eb376e853a0a1c2b1ac838cb9d61f385100",  # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_typedb_protocol
```

Comment:
Will need to update it once the protocol is released, can be done after the merge.